lightbeam/util.py

```diff
@@ -23,6 +23,7 @@ def merge_dicts(user, default):
 # Converts (for example) `LocalEducationAgencies` to `LocalEducationAgency`; `students` to `student`; etc.
 def singularize_endpoint(endpoint):
     if endpoint[-3:]=="ies": return endpoint[0:-3] + "y"
+    elif endpoint=="people": return "person"
```

Comment:
Who could have ever predicted that such a naming convention would come to bite the Ed-Fi Alliance? 😆